include/flatbuffers/reflection.h

```diff
@@ -47,29 +47,31 @@ inline bool IsLong(reflection::BaseType t) {
 inline size_t GetTypeSize(reflection::BaseType base_type) {
   // This needs to correspond to the BaseType enum.
   static size_t sizes[] = {
-    0, // None
-    1, // UType
-    1, // Bool
-    1, // Byte
-    1, // UByte
-    2, // Short
-    2, // UShort
-    4, // Int
-    4, // UInt
-    8, // Long
-    8, // ULong
-    4, // Float
-    8, // Double
-    4, // String
-    4, // Vector
-    4, // Obj
-    4, // Union
-    0, // Array. Only used in structs. 0 was chosen to prevent out-of-bounds errors.
+    0,  // None
+    1,  // UType
+    1,  // Bool
+    1,  // Byte
+    1,  // UByte
+    2,  // Short
+    2,  // UShort
+    4,  // Int
+    4,  // UInt
+    8,  // Long
+    8,  // ULong
+    4,  // Float
+    8,  // Double
+    4,  // String
+    4,  // Vector
+    4,  // Obj
+    4,  // Union
+    0,  // Array. Only used in structs. 0 was chosen to prevent out-of-bounds
+        // errors.
```

Comment:
This has the same formatting as the earlier resolved conversation, but do we want to have each comment on a line?

Reply:
Yes, this one is fine to keep.